EarthBnB/src/main/java/com/Earthbb/Map/MapController.java

```diff
@@ -0,0 +1,45 @@
+package com.Earthbb.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+
+@CrossOrigin(origins = { "http://localhost:4200" })
+@RestController
+@RequestMapping("/api/v1/map")
+public class MapController {
+
+    @Autowired
+    private MapRepository mapRepository;
+
+    @GetMapping
+    public List<Map> list() { return mapRepository.findAll();}
+
+    @GetMapping("{id}")
+    public Map get(@PathVariable("id") long id) { return mapRepository.getOne(id);}

```

Comment:
does this need "/{id}" or is "{id}" fine? cause your request mapping doesnt have and '/' on the end, so if you use GET localhost:8080/api/v1/map/# ... would that fail or does it put on on the end when you do that so it doesn matter?

Reply:
Esp cause delete and put have it as "/{id}" so shouldnt get as well?